test/controllers/restaurants_controller_test.rb

```diff
@@ -17,7 +17,7 @@ class RestaurantsControllerTest < ActionDispatch::IntegrationTest
 
   test 'should create restaurant' do
```

Comment:
Please test these using RSpec rather than Rails' tests.

I won't force you to write tests for every single scenario in the restaurants model and controller, but what you DO test should be in RSpec

Reply:
Those tests are autogenerated
I did some RSpec tests, I'll rewrite those autogenerated tests using RSpec
Thanks